cfme/infrastructure/cluster.py

```diff
@@ -117,7 +117,12 @@ class All(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self):
-        self.parent_view.navigation.select('Compute', 'Infrastructure', 'Clusters')
+        try:
```

Comment:
Is this version dependent? If so I think it would be better to navigate based the version and not just catch the exception.